<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞–Ω–∏–µ –ø—Ä–æ—Ñ–∏–ª—è</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #ffffff;
}
QLineEdit {
    border: 2px solid #ced4da;
    border-radius: 20px;
    padding: 12px 20px;
    font-size: 16px;
    background-color: #f8f9fa;
}
QLineEdit:focus {
    border-color: #ff3333;
    background-color: white;
}
QPushButton#backButton {
    background-color: transparent;
    border: none;
    min-width: 45px;
    min-height: 45px;
}
QPushButton#backButton:hover {
    background-color: #f0f0f0;
    border-radius: 22px;
}
QPushButton#saveButton {
    background-color: #ff3333;
    color: white;
    font-weight: bold;
    border: none;
    border-radius: 20px;
    padding: 10px 24px;
    font-size: 14px;
}
QPushButton#saveButton:hover {
    background-color: #ff5252;
}
QLabel {
    color: #212529;
}
QLabel.fieldLabel {
    color: #ff3333;
    font-weight: bold;
    font-size: 14px;
    margin-bottom: 8px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topBarLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="margin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/icons/arrow.png</normaloff>:/icons/icons/arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>927</width>
        <height>600</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="margin">
        <number>40</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="photoLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="photoLeftSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="photoLabel">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>border: 2px dashed #ced4da; border-radius: 100px; background-color: #f8f9fa;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="text">
            <string>üë§</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="photoRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="fieldsLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>–ò–º—è:</string>
           </property>
           <property name="styleSheet">
            <string>color: #ff3333; font-weight: bold; font-size: 14px; margin-bottom: 8px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firstNameLineEdit">
           <property name="placeholderText">
            <string>–í–≤–µ–¥–∏—Ç–µ –≤–∞—à–µ –∏–º—è...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lastNameLabel">
           <property name="text">
            <string>–§–∞–º–∏–ª–∏—è:</string>
           </property>
           <property name="styleSheet">
            <string>color: #ff3333; font-weight: bold; font-size: 14px; margin-bottom: 8px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lastNameLineEdit">
           <property name="placeholderText">
            <string>–í–≤–µ–¥–∏—Ç–µ –≤–∞—à—É —Ñ–∞–º–∏–ª–∏—é...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="positionLabel">
           <property name="text">
            <string>–î–æ–ª–∂–Ω–æ—Å—Ç—å:</string>
           </property>
           <property name="styleSheet">
            <string>color: #ff3333; font-weight: bold; font-size: 14px; margin-bottom: 8px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="positionLineEdit">
           <property name="placeholderText">
            <string>–í–≤–µ–¥–∏—Ç–µ –≤–∞—à—É –¥–æ–ª–∂–Ω–æ—Å—Ç—å...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>

